app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -14,11 +14,14 @@ import java.net.URL
 class TransparentTileOWM(private val tileType: String):TileProvider {
     private val opacityPaint = Paint()
 
+    init {
+        setOpacity(50)
+    }
     /**
      * Sets the desired opacity of map [Tile]s, as a percentage where 0% is invisible and 100% is completely opaque.
      * @param opacity The desired opacity of map [Tile]s (as a percentage between 0 and 100, inclusive)
      */
-    fun setOpacity(opacity: Int) {
+    internal fun setOpacity(opacity: Int) {
```

Comment:
Updated visibility modifiers as recommended from: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces